File: MD_sample/Python_imperative/print_snap.py
from md import Npt

def print_snap(isnap,q):
    fname = "snapshot" + str(isnap).zfill(5) + ".xyz"
    out = open(fname,'w')
    print >> out, Npt
    print >> out, "######"
    for i in range (Npt):
        print >> out, "C ",
        for k in range (3):
            print >> out, " ", q[i].xx[k],
        print >> out
    out.close()

    

File: MD_sample/F90/print_snap.f90
subroutine print_snap(isnap)
  use md
  implicit none
  integer(di):: isnap, i
  character(256):: fname
  write(fname, 10) isnap
10 format("snapshot", I5.5, ".xyz")
  open(unit=25, file=fname, ACTION="WRITE")
  write(25,*) Npt
  write(25,*) "Header"
  do i=1_di, Npt
     write(25,20), "C ", q(i)%xx(1:3)
  end do
20 format(A2, 3(1X, ES12.5))
  close(25)
  return
end subroutine print_snap
